x/likenft/keeper/nft_helper.go

```diff
@@ -38,3 +39,15 @@ func (k Keeper) resolveValidClaimPeriod(ctx sdk.Context, classId string, classDa
 
 	return nil, nil
 }
+
+func SortClaimPeriod(claimPeriods []*types.ClaimPeriod, descending bool) []*types.ClaimPeriod {
+	// Sort the claim periods by start time
+	sort.Slice(claimPeriods, func(i, j int) bool {
```

Comment:
can we sort by time then price (if same timestamp?), for discount for specific users